proto/p4/config/p4info.proto

```diff
@@ -21,13 +21,11 @@ import "google/protobuf/any.proto";
 package p4.config;
 
 message P4Info {
-  repeated HeaderField header_fields = 1;
```

Comment:
@wmohsin I believed we discussed this today during our talk. This mapping would be the responsibility of the p4c backend for the target. We can discuss this more on Friday or by email with @teverman if there are more concerns.